tripal_analysis_expression/tripal_analysis_expression.module

```diff
@@ -111,62 +111,106 @@ function tripal_analysis_expression_menu()
     'type' => MENU_CALLBACK,
   ];
 
-  $items['tripal_analysis_expression/pvalue/cvterm/%'] = [
+  $items['tripal_analysis_expression/pvalue/cvterm/%/%'] = [
     'title' => 'Expression Analysis CVterm',
     'description' => 'P-value CV terms.',
     'access arguments' => ['access content'],
-    'page arguments' => [3],
+    'page arguments' => [3, 4],
     'page callback' => 'tripal_analysis_expression_cvterm',
   ];
   return $items;
 }
 
-function tripal_analysis_expression_cvterm($cvterm_id)
+/**
+ *  This function creates two tables on a custom page, linked to from
+ *  the keyword on the significant expression page. The first table lists
+ *  CVterm name, definition, and vocabulary, and the second lists associated
+ *  features with p-values. A pager appears when there are greater than
+ *  twenty features.
+ *
+ * @param $cvterm_id
+ * @param $organism_id
+ *
+ * @return array
+ * @see \theme() for paramaters instructions
+ *
+ * @ingroup tripal_analysis_expression
+ *
+ */
+
+function tripal_analysis_expression_cvterm($cvterm_id, $organism_id)
 {
+  // adding upper table which contains name, definition, and vocabulary
   $cv_term = chado_query('select cv.name as cv_name, cv.definition as cv_definition, db.name as db_name
                 from {chado.cvterm} cv
                 join {chado.dbxref} dbx on dbx.dbxref_id=cv.dbxref_id
                 join {chado.db} db on db.db_id=dbx.db_id
                 where cvterm_id=:cvterm_id',
     [':cvterm_id' => $cvterm_id])->fetchObject();
+  $organism = chado_query('select * from chado.feature where organism_id=:organism_id',
+    [':organism_id' => $organism_id])->fetchObject();
+  $cvterm = chado_query('select * from chado.cvterm where cvterm_id=:cvterm_id',
```

Comment:
This query doesn't look like it's needed since you already have the info in `$cv_term` from the query above it.